pages/api/openPR.js

```diff
@@ -249,38 +249,36 @@ export default async (req, res) => {
         parseProjectName(values),
         getSubmissionFiles(values, nomineeJSON)
       )
-    ).then((results) => {
```

Comment:
There is nothing wrong with this code: it says that once all promises have been settled, `then` and only then execute the following code.